docs/machine-learning/tutorials/object-detection-model-builder.md

```diff
@@ -255,6 +255,21 @@ Two projects are created as a result of the training process.
     Top: 89.453415, Left: 481.95343, Right: 724.8073, Bottom: 388.32385, Label: Stop-Sign, Score: 0.99539465
     ```
 
+    > [!NOTE]
+    > The bounding box coordinates are normalized for a width of 800 pixels and height of 600 pixels. To scale the coordinates for your image, you need to:
```

Comment:
nit: "...a width of 800 pixels and **a** height of 600 pixels."

Reply:
Done